From f27ba380bdfe260183e780948e30157cd17b4e27 Mon Sep 17 00:00:00 2001
From: "Jan.Hannig" <jan.hannig@irisgmbh.de>
Date: Wed, 11 May 2022 14:27:23 +0200
Subject: [PATCH] [RDPHOEN-958] Remove CONFIG_CMD_BOOTEFI for
 imx8mp_evk_defconfig

As the function "efi_set_bootdev" called within this compiler switch
causes the error msg "**Bad device specification mmc 2#linuxboot_a **".

Signed-off-by: Jan.Hannig <jan.hannig@irisgmbh.de>
---
 configs/imx8mp_evk_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/imx8mp_evk_defconfig b/configs/imx8mp_evk_defconfig
index 82d1281f5b..0198e0beeb 100644
--- a/configs/imx8mp_evk_defconfig
+++ b/configs/imx8mp_evk_defconfig
@@ -43,6 +43,7 @@ CONFIG_SYS_PROMPT="u-boot=> "
 # CONFIG_CMD_IMPORTENV is not set
 CONFIG_CMD_ERASEENV=y
 # CONFIG_CMD_CRC32 is not set
+# CONFIG_CMD_BOOTEFI is not set
 # CONFIG_BOOTM_NETBSD is not set
 CONFIG_CMD_CLK=y
 CONFIG_CMD_FUSE=y
-- 
2.20.1

